modules/networkfirewall-dashboard/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.70.9"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+resource "lightstep_dashboard" "aws_networkfirewall_dashboard" {
+  project_name   = var.lightstep_project
+  dashboard_name = "AWS Network Firewall"
+
+  chart {
+    name = "Passed Packets / Dropped Packets "
+    rank = "0"
+    type = "timeseries"
+
+    query {
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = <<EOT
+with
+  a = metric aws.networkfirewall.passed_packets_sum | delta | group_by [], sum;
+  b = metric aws.networkfirewall.passed_packets_count | delta | group_by [], sum;
+join (a / b), a = 0, b = 0
+EOT
+    }
+
+    query {
+      query_name   = "b"
+      display      = "line"
+      hidden       = false
+      query_string = <<EOT
+with
+  a = metric aws.networkfirewall.dropped_packets_sum | delta | group_by [], sum;
+  b = metric aws.networkfirewall.dropped_packets_count | delta | group_by [], sum;
+join (a / b), a = 0, b = 0
+EOT
+    }
+
+  }
+
+  chart {
+    name = "Received Packets Count"
+    rank = "1"
+    type = "timeseries"
+
+    query {
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = <<EOT
+with
+  a = metric aws.networkfirewall.received_packet_count_sum | delta | group_by [], sum;
+  b = metric aws.networkfirewall.received_packet_count_count | delta | group_by [], sum;
+join (a / b), a = 0, b = 0
+EOT
+    }
+
+  }
+
+  chart {
+    name = "Packets / Bytes Out"
+    rank = "2"
+    type = "timeseries"
+
+    query {
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = <<EOT
+with
```

Comment:
The name `Packets / Bytes Out` seems unrelated to the actually `timeseries` that are being joined.